api/explore/explore.js

```diff
@@ -30,13 +31,38 @@ async function innerSearch(searchString, amount, request) {
   return values.items;
 }
 
-async function GetBoxes(size, dateString, request, amount) {
+function mealByTime(timeString) {
+  let nextMeal = '';
+  let greet = '';
+  const format = 'HH:mm:ss';
+  const time = moment(timeString, format);
+
+  if (time.isBetween(moment('05:00:00', format), moment('09:59:59', format), null, '[]')) {
+    nextMeal = 'Breakfast';
+    greet = 'Good Morning, try breakfast recipes:';
```

Comment:
Where is this displayed, not sure we should say Good Morning if it's in the middle of the page

Reply:
If it won't be at the top of the page, we will be able choose not to have the greet.
I think it should be at the top though...